spec/category_spec.rb

```diff
@@ -828,6 +828,12 @@
         expect(@dv[:yoda]).to eq(666)
       end
 
+      it "assigns a new category at a new index " do
+        @dv.add_category :abc
+        @dv.send(:set, :xtbr, :abc)
+        expect(@dv[:xtbr]).to eq(:abc)
```

Comment:
It would be good if you also test multiple assignment like `dv.set [:a, :v], :abc`